control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -60,13 +61,20 @@ var (
 	ErrNoDeadLetterSinkURI = errors.New("no dead letter sink URI resolved")
 )
 
-type schedulerFunc func(s string) scheduler.Scheduler
+type Scheduler struct {
+	scheduler.Scheduler
+	SchedulerConfig
```

Comment:
Config values are already stored in `scheduler.Scheduler`...why do we need it separately in this type?

Reply:
`scheduler.Scheduler` is the interface with only the `Schedule` method.